concepticondata/concepticon.tsv

```diff
@@ -3788,3 +3788,4 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3802	FERN	Agriculture and vegetation	Any member of a large group of vascular plants in the family Filices who have neither flowers nor seeds.	Person/Thing	
 3803	RASPBERRY	Agriculture and vegetation	The edible, berry-like fruit of a multitude of plant species in the genus Rubus of the family Rosaceae, most typically red, small and sweet.	Person/Thing	
 3804	GRAPEFRUIT OR POMELO	Agriculture and vegetation	The fruit of the pomelo tree, Citrus maxima, of the family Rutaceae, or the fruit of the grapefruit tree, Citrus x paradisi, a citrus hybrid which originated from crossing pomelos and oranges. Due to the recent origin of the grapefruit and the close genetic relatedness between both fruits, they are often not differentiated lexically.	Person/Thing	
+3805	SHY OR ASHAMED	Emotions and values	The feeling of being uncomfortable with having attention drawn to oneselve or when feeling shame or guilt.	Property	
```

Comment:
small typo: oneself
I think "having attention drawn to oneself" is not wrong, but to me it seems a bit clunky in this sentence. Perhaps just "The feeling of being uncomfortable with attracting attention or feeling shame or guilt"?